blinkpy/auth.py

```diff
@@ -205,7 +172,7 @@ def query(
             code = None
             reason = None
             try:
-                code = response.status_code
+                code = response.status
```

Comment:
Yes - `aiohttp` response structure is .status rather than `.status_code`
You will also notice the '.text' (and other streams) are async functions that return strings rather than strings.